build/params_testground.go

```diff
@@ -98,3 +98,5 @@ var (
 
 	Devnet = true
 )
+
+const BootstrapPeerThreshold = 4
```

Comment:
this should probably be 1 for testground.

Reply:
hi @vyzo 
now when BootstrapPeerThreshold = 1 test fail

Reply:
now test ok